Rakefile

```diff
@@ -14,7 +14,9 @@ task :check_urls do
                 # Ignore pulls/branches as these do not translate to raw content
                 %r{github\.com/hmcts/(?=.*(?:pull|tree|commit))},
                 # This is a url that's generated each time we build the html by tech-docs-gem but does not exist
-                %r{https://github.com/hmcts/hmcts.github.io/blob/source/source/search/index.html}
+                %r{https://github.com/hmcts/hmcts.github.io/blob/source/source/search/index.html},
+                # This handles new files that haven't been merged to master branch yet for this repo in a PR
+                %r{(?=.*hmcts.github.io)(?=.*github)}
```

Comment:
Could you explain how this works more? I don't understand it.

Ignore everything *hmcts.github.io and *github?

Reply:
More of a workaround for now hopefully:

When a new page is added, easier example https://github.com/hmcts/ops-runbooks/pull/290, the link checker isn't happy because a new HTML link is auto-built by middleman build for the new page, but because we haven't published yet the page doesn't exist


The links it creates are a generic html website link, and a raw user content link, both these have the hmcts.github.io and the repo name, but I didn't want to maybe miss any other references
No doubt it can be done better with more time, but was blocking a few people and this was the simplest way I could see to unblock quickly

Reply:
won't this mean that any internal links fail to the site fail? Although they should be doing relative links and not absolute

Reply:
This change only affects external linking - the number of internal link checks stays the same with/without the change